docs/releasing.md

```diff
@@ -119,28 +144,19 @@ For each minor or major release, two jobs need to be created :
 ### Update Metal3-dev-env
 
 Metal3-dev-env variables need to be modified. After a major or minor release,
-the new minor version (that follows CAPI versioning) should point to main for
-CAPM3 and the released version should point to the release branch.
+the new minor version should point to main for IPAM and the released version
+should point to the release branch.
 
-### Update the image of CAPM3 in the release branch
+WARNING: this breaks metal3-dev-env (CI included) until dev-env and IPAM/CAPM3
+repositories are in sync.
 
-If you just created a release branch (i.e. minor version release), you should
-modify the image for CAPM3 deployment in this branch to be tagged with the
-branch name. The image will then follow the branch.
+### Update testing matrix documentation
 
-### Create a new tag for Ironic image
+Update [testing matrix documentation](https://github.com/metal3-io/metal3-docs/pull/320)
```

Comment:
This should probably point to a file upstream when the PR is merged.

Reply:
or the commit hash

Reply:
Yes, it'll point to real document link when it gets merged. For now, it points to PR.